vault/src/system.rs

```diff
@@ -98,7 +98,11 @@ pub struct VaultServiceConfig {
 
     /// The currency to use for the collateral, e.g. "DOT" or "KSM".
     #[clap(long, parse(try_from_str = parse_collateral_currency))]
-    pub currency_id: Option<CurrencyId>,
+    pub collateral_currency_id: Option<CurrencyId>,
```

Comment:
I think there is some clap option to require another arg if you pass this one, something like `#[clap(long, requires("wrapped-currency-id"), parse(try_from_str = parse_collateral_currency))]`, or possibly `requires = "wrapped-currency-id"`. Could you try that?

Reply:
Used `require`, but it seems a bit buggy. It logs the same missing argument three times now:
```
error: The following required arguments were not provided:
    --collateral-currency-id <COLLATERAL_CURRENCY_ID>
    --collateral-currency-id <COLLATERAL_CURRENCY_ID>
    --collateral-currency-id <COLLATERAL_CURRENCY_ID>
```

Reply:
Hmm weird. Probably fine for now though